src/EnergyPlus/FaultsManager.cc

```diff
@@ -206,7 +265,7 @@ namespace FaultsManager {
 		// na
 
 		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-		static bool RunMeOnceFlag( false );
+		static bool RunFaultMgrOnceFlag( false );
```

Comment:
@rongpengzhang I think we want this and other statics to be moved up into an anonymous namespace and then add a clear_state function to reset them?  @Myoldmopar @EnergyArchmage ?